app/Console/Kernel.php

```diff
@@ -22,7 +22,7 @@ class Kernel extends ConsoleKernel {
         '\App\Console\Commands\RosterRemovalWarn',
         '\App\Console\Commands\VatsimAtcBookingSync',
         '\App\Console\Commands\VATUSAEventsUpdate',
-        '\App\Console\Commands\UploadTrainingTickets'
+        '\App\Console\Commands\UploadTrainingTickets',
```

Comment:
What changed here? Look out of scope for this PR.

Reply:
Sorry, accidentally left in that comma.